swift/src/Ice/LocalException.swift

```diff
@@ -1436,23 +1385,6 @@ open class MemoryLimitException: MarshalException {
     }
 }
 
-/// This exception is raised when a string conversion to or from UTF-8 fails during marshaling or unmarshaling.
-open class StringConversionException: MarshalException {
```

Comment:
That's a C++ decoding exception that doesn't apply to Swift and MATLAB since they perform their own string decoding without throwing this exception.